mumlib/src/error.rs

```diff
@@ -42,3 +42,42 @@ impl fmt::Display for ChannelIdentifierError {
         }
     }
 }
+
+pub enum ConfigError {
+    InvalidConfig,
+    TOMLErrorSer(toml::ser::Error),
+    TOMLErrorDe(toml::de::Error),
+
```

Comment:
Why this newline?

Reply:
Grouping. The first group is "the config looks bad", the second group are more file system errors.